components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.m

```diff
@@ -178,12 +180,70 @@ - (void)setOuterHighlightColor:(UIColor *)outerHighlightColor {
   _outerLayer.fillColor = _outerHighlightColor.CGColor;
 }
 
+- (void)setTitleFont:(UIFont *)titleFont {
+  _titleFont = titleFont;
+
+  [self updateTitleFont];
+}
+
+- (void)updateTitleFont {
+  // If we have a custom font apply it to the label.
+  // If not, fall back to the Material specified font.
+  if (_titleFont) {
+    // If we are automatically adjusting for Dynamic Type resize the font based on the text style
+    if (_mdc_adjustsFontForContentSizeCategory) {
+      _titleLabel.font =
+          [_titleFont mdc_fontSizedForMaterialTextStyle:kTitleTextStyle
+                                   scaledForDynamicType:_mdc_adjustsFontForContentSizeCategory];
+    } else {
+      _titleLabel.font = _titleFont;
+    }
+  } else {
+    if (_mdc_adjustsFontForContentSizeCategory) {
+      _titleLabel.font = [UIFont mdc_preferredFontForMaterialTextStyle:kTitleTextStyle];
+    } else {
+      _titleLabel.font = [UIFont mdc_standardFontForMaterialTextStyle:kTitleTextStyle];
+    }
+  }
+
+  [self setNeedsLayout];
+}
+
 - (void)setTitleColor:(UIColor *)titleColor {
   _titleColor = titleColor;
 
   _titleLabel.textColor = titleColor;
 }
 
+- (void)setBodyFont:(UIFont *)bodyFont {
+  _bodyFont = bodyFont;
+
+  [self updateBodyFont];
+}
+
+- (void)updateBodyFont {
+  // If we have a custom font apply it to the label.
+  // If not, fall back to the Material specified font.
+  if (_bodyFont) {
+    if (_mdc_adjustsFontForContentSizeCategory) {
+      // If we are automatically adjusting for Dynamic Type resize the font based on the text style
+      _bodyLabel.font =
+          [_bodyFont mdc_fontSizedForMaterialTextStyle:kBodyTextStyle
+                                  scaledForDynamicType:_mdc_adjustsFontForContentSizeCategory];
+    } else {
+      _bodyLabel.font = _bodyFont;
+    }
+  } else {
+    if (_mdc_adjustsFontForContentSizeCategory) {
+      _bodyLabel.font = [UIFont mdc_preferredFontForMaterialTextStyle:kBodyTextStyle];
```

Comment:
we want to use the singleton's value here too

Reply:
Done.